Resources/Textures/Clothing/Eyes/Glasses/aviators.rsi/meta.json

```
@@ -0,0 +1,26 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Sprites created by MrGreen06 (Discord) ported ingame by PursuitinAshes.",
+    "size": {
+      "x": 32,
+      "y": 32
+    },
+    "states": [
+      {
+        "name": "Icon"
```

```suggestion
        "name": "icon"
```
Believe that should fix the issue as it auto-uses "icon" (lowercase) for the icon.